spec/spec_helper.rb

```diff
@@ -41,6 +43,21 @@
   exit 1
 end
 
+Capybara.register_driver :selenium_remote_chrome do |app|
+  chrome_options = Selenium::WebDriver::Chrome::Options.new(args: ['--no-sandbox', '--disable-gpu',
+                                                                   '--window-size=1400,1400'])
+  chrome_options.add_argument('--headless') unless ENV.fetch('DISABLE_HEADLESS_UI_TESTING', nil) == 'true'
+  Capybara::Selenium::Driver.new(app, browser: :remote, url: 'http://localhost:9515', capabilities: [chrome_options])
```

Comment:
`localhost` doesn't work for me, so please change this back to `host.docker.internal`.

Or, you can help debug the following error:

```
Failure/Error: TCPSocket.open(conn_addr, conn_port, @local_host, @local_port)

          Errno::EADDRNOTAVAIL:
            Failed to open TCP connection to localhost:9515 (Cannot assign requested address - connect(2) for "localhost" port 9515)
```

Reply:
@anoverga17 ah never mind, I rebuilt the docker containers and then the tests worked fine with `localhost`.